x-pack/plugins/security_solution/public/timelines/components/timeline/body/column_headers/column_header.tsx

```diff
@@ -4,26 +4,49 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { EuiContextMenu, EuiContextMenuPanelDescriptor, EuiIcon, EuiPopover } from '@elastic/eui';
```

Comment:
Thank you for matching the functionality of EuiDatagrid so user do not get lost with different way of doing thing. We appreciate that.